docs/release/itb-testing.md

```diff
@@ -25,50 +28,85 @@ The commands can be run as-is from within the `osgitb` directory from git.
         :::console
         ansible current -i inventory -f 20 -o -m command -a 'cat /etc/redhat-release'
 
-2. Check HTCondor versions for all HTCondor hosts:
+1. Check the date and time on all hosts (in case NTP stops working):
 
         :::console
-        ansible condor -i inventory -f 20 -o -m command -a 'rpm -q condor'
+        ansible current -i inventory -f 20 -o -m command -a 'date'
+
+1. Check software versions for certain hosts (e.g., for the `condor` package on hosts in the `worksers` group):
+
+        :::console
+        ansible workers -i inventory -f 20 -o -m command -a 'rpm -q condor'
 
-3. Obtain the NVR of the HTCondor prerelease build from OSG to test.  Do this by talking to Tim&nbsp;T. and checking
-   Koji.  The expectation is that the HTCondor prerelease build will be in the development repository (or
-   upcoming-development).
 
-## Install HTCondor Prerelease
+## Installing HTCondor Prerelease
+
+Use this section to install a new version of HTCondor, specifically a prerelease build from the development or
+upcoming-development repository, on the test hosts.
+
+1. Obtain the NVR of the HTCondor prerelease build from OSG to test.  Do this by talking to Tim&nbsp;T. and checking
+   Koji.
 
 1. Shut down HTCondor and HTCondor-CE on prerelease machines:
 
         :::console
-        ansible condordev -i inventory -bK -f 20 -m service -a 'name=condor-ce state=stopped' -l 'itb-ce*'
-        ansible condordev -i inventory -bK -f 20 -m service -a 'name=condor state=stopped'
+        ansible 'testing:&ces' -i inventory -bK -f 20 -m service -a 'name=condor-ce state=stopped'
+        ansible 'testing:&condor' -i inventory -bK -f 20 -m service -a 'name=condor state=stopped'
```

Comment:
What's the `:&` notation?

Reply:
Just a little bit of Ansible awesomeness! It means “take the intersection of the two groups”, and in this case is a much better way to express the set of hosts being targeted instead of a group plus a limit option at the end. (IMO)